src/styles/main.scss

```diff
@@ -0,0 +1,177 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+body {
+  color: #7b7b7b;
+  background-color: #ddd;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 100;
+  font-size: 20px;
+}
+
+textarea {
+  border: none;
+  width: 100%;
+  height: 1.5em;
+  resize: none;
+  outline: none;
+  font-size: 1rem;
+  padding-top: 2px;
+  font-family: 'Roboto', sans-serif;
```

Comment:
If you are reusing the same font family in all of your classes, it is best to add it to the `*` tag.

Reply:
@tkz96 Yeah i will fix that :+1:  :handshake: